### ISSUE TITLE
Done button is not functional

### ISSUE THREAD

**Issue by jwall704** [JohnCoates/Aerial#494]
Steps:
1. Launch System Preferences
2. Select Desktop & Screen Saver
3. Select Aerial screen saver
4. Select Screen Saver Options
5. Make necessary changes
6. Click Done

Results:
No response when clicking done. Options UI remains in place.

Workaround:
Force Quit System Preferences

**Comment by frenchfalcon** [comment]
Exactly and I noticed that previews won't work when clicking done.
10.14

**Comment by glouel** [comment]
See here for a fix in the meantime : https://github.com/JohnCoates/Aerial/issues/463#issuecomment-423932693

**Comment by joshers2005** [comment]
Ah, will they ever come back? Those were my favorites. Until then, I am sticking back to the old release.

**Comment by glouel** [comment]
Please check the comment I linked about that. Short answer, maybe.

**Comment by esetnik** [comment]
@joshers2005 @glouel I see no reason why we can't merge both new and old lists to create a comprehensive set of all 4k assets. I'm not really concerned with even going as far as hard-coding the asset links instead of downloading a payload from Apple. I don't see what the difference is between hard-coding a payload link and hardcoding the media links. Also this would enable us to open up the UI to allow users to provide their own supplementary media links for any 4k content that's publicly available. We could even accept PRs for known good 4k content from third-party sources if we wanted to. My suggestion would be to create and commit our own manifest.json with a big array of media links that are tagged with the quality, format, description, features, etc.

**Comment by glouel** [comment]
@esetnik  In that case, those are 1080p only. Cf my linked comment, the tvOS11 JSON (from your branch) only had 2 videos that weren't included already in the tvOS12 JSON we use now.

**Comment by esetnik** [comment]
The idea to dynamically pull the media was based on the concept that apple changes the media list dynamically between tvOS releases. This has not been the case to my knowledge. Instead, each year they have changed the url / file type of the media list leading to breakage and extra work (like untarring). I think it’s better we just create our own list of the assets themselves in the format Aerial best consumes.

**Comment by glouel** [comment]
Have you checked this comment I made about exactly all of this ?  : 
https://github.com/JohnCoates/Aerial/issues/463#issuecomment-424744678

**Comment by joshers2005** [comment]
Yes. I am primary pissed that over half of the videos got removed. Unless the bugs are doing harm for my iMac's HD, I am keeping the previous version I download yesterday that has all the extra media. I will update the screen saver only when my favorite cities I love come back as I feel the space videos are not as interesting.

**Comment by esetnik** [comment]
> Have you checked this comment I made about exactly all of this ? :
> [#463 (comment)](https://github.com/JohnCoates/Aerial/issues/463#issuecomment-424744678)

@glouel perfect!
